src/components/VideoJS/VideoJS.module.scss

```diff
@@ -21,7 +22,6 @@
 
   &.isFit video {
     object-fit: contain !important;
-    background: #000000;
```

Comment:
Oh, I see what you mean. Well, the reason I thought this would be a useful change is because of how difficult it is right now to spot the boundaries of a video tile when proportional scaling is enabled. The black bars above and underneath the video blend with the dotted background and you need to look hard to notice where can you grab the video feed to resize it. 

How about we drop this change for now and circle back to the idea of introducing a more "native-like" full-screen mode? Once we have that implemented, then we can implement this background change.